src/libm/sleefsimddp.c

```diff
@@ -268,7 +277,7 @@ EXPORT CONST vdouble xsin(vdouble d) {
   d = vmla_vd_vd_vd_vd(dqh, vcast_vd_d(-PI_C), d);
   d = vmla_vd_vd_vd_vd(dql, vcast_vd_d(-PI_C), d);
   d = vmla_vd_vd_vd_vd(vadd_vd_vd_vd(dqh, dql), vcast_vd_d(-PI_D), d);
-  
+
```

Comment:
Please don't change this line, we don't have a coding standard that enforces to remove lines with spaces.